common/changes/@itwin/core-frontend/andremig-highlight-clip-edge_2023-10-10-13-28.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-frontend",
+      "comment": "Added ClipHighlight API functionality, and 'fdt clip highlight' keyin",
```

Comment:
Shouldn't there be a changelog for the frontend-devtools package too? That's where the key-in resides, not core-frontend.